include/quicr/message_buffer.h

```diff
@@ -14,18 +14,22 @@ class uintVar_t
   uintVar_t() = default;
   constexpr uintVar_t(const uintVar_t&) = default;
   constexpr uintVar_t(uintVar_t&&) = default;
-  constexpr uintVar_t(uint64_t v)
-    : _value{ v }
+  constexpr uintVar_t(uint64_t value)
+    : _value{ value }
   {
-    assert(v < 0x1ull << 61);
+    if (value >= 0x1ull << 61)
+      throw std::runtime_error("Max value cannot be exceeded: " +
```

Comment:
We can fix this later, but weren't we throwing custom classes so that we can catch/distinguish them?